pkgs/top-level/all-packages.nix

```diff
@@ -309,6 +309,8 @@ with pkgs;
 
   buildMaven = callPackage ../build-support/build-maven.nix {};
 
+  buildIncremental = callPackage ../build-support/build-incremental.nix {};
```

Comment:
I think something like `incrementalBuildTools` would be more suitable since this is not really a package by itself.